src/fsharp/vs/ServiceDeclarations.fs

```diff
@@ -1365,8 +1365,18 @@ type FSharpAccessibility(a:Accessibility, ?isProtected) =
 
 /// An intellisense declaration
 [<Sealed>]
-type FSharpDeclarationListItem(name: string, nameInCode: string, fullName: string, glyph: FSharpGlyph, info, isAttribute: bool, accessibility: FSharpAccessibility option,
-                               kind: CompletionItemKind, isOwnMember: bool, priority: int) =
+type FSharpDeclarationListItem
+    ( name          : string
```

Comment:
I'm putting the commas at the end of each parameter, do you want me to adjust something else?

The reason I changed that is that I want to get a good overview of the parameters, and the original code doesn't allow me to scan that easily.

Motive for this formatting:
* vertical alignment helps scanning names and/or types, and having comma at the front doesn't clutter the type name / easier to align / scan
* when someone eventually adds documentation to those important items, it will have to be on each line
* long lines are more difficult to comprehend, I don't think there is good reason to keep them all on same line or several jagged/long lines